api/views_export.py

```
@@ -0,0 +1,24 @@
+import json
+from django.http import JsonResponse
+from django.core.serializers.json import DjangoJSONEncoder
+from patients_core.models import Patient
+from diab_encounters.models import Encounter
+from diab_labs.models import LabResult
+from diab_medications.models import MedicationOrder
+from ai_summarizer.models import AISummary
+
+def export_patient(request, pk):
```

Ruff (0.12.2)

Missing return type annotation for public function `export_patient`

(ANN201)